Dockerfile.arm64

```diff
@@ -0,0 +1,78 @@
+ARG PY_VERSION_DEFAULT=3.9
+
+FROM dokken/centos-stream-9 as base
+
+USER root
+
+ARG PY_VERSION_DEFAULT
+ARG MEMGRAPH_RPM
```

Comment:
Why is this an argument, and not the image downloaded from the Memgraph download page?

Reply:
This should be changed, this PR for now is something that works only locally and with manually building the RPM package on an ARM machine. 
This isn't something that should be merged as is, I only wanted to make and show changes that are necessary to make it work on Centos 9/ARM64.